chat/admin.py

```diff
@@ -0,0 +1,3 @@
+# from django.contrib import admin
```

Comment:
Is it needed?

Reply:
> Please think of a test to verify what ever exist here is working. Please refer last tests talk

Added a simple test in [4cf4d57](https://github.com/redhat-beyond/JobSeeker/pull/43/commits/4cf4d5762c6aa801dab4693f09e6a0472c377646).


> Is it needed?

That line is commented-out so *Flake8* could approve.
Do you prefer to remove this line entirely?